foreman.te

```diff
@@ -236,7 +239,11 @@ optional_policy(`
 ')
 
 tunable_policy(`httpd_run_foreman', `
-    allow httpd_t passenger_tmp_t:sock_file write;
+    manage_dirs_pattern(httpd_t, passenger_tmp_t, passenger_tmp_t)
+    manage_files_pattern(httpd_t, passenger_tmp_t, passenger_tmp_t)
+    manage_sock_files_pattern(httpd_t, passenger_tmp_t, passenger_tmp_t)
```

Comment:
I assume `manage_sock_files_pattern` replaces `allow httpd_t passenger_tmp_t:sock_file write;`, is that correct?

Reply:
Exactly, see here: https://github.com/fedora-selinux/selinux-policy/blob/rawhide/policy/support/file_patterns.spt#L373-L376

Grep the repo further to find `policy/support/obj_perm_sets.spt` file holding another macro with all the actions which includes the `write` operation. Just keep in mind when looking up macros that we target CentOS7 and git repo for that is internal (or use centos7 SRPM instead). There is lot of development already.

Reply:
To further explain, passenger_t domain is already allowed all above, but httpd_t is not. We want to keep both on par when the foreman_run_httpd flag is turned on. It was a bit behind.